opentelemetry-sdk/CHANGELOG.md

```diff
@@ -24,6 +24,43 @@
      Migration Guidance: 
         - These methods were intended for log appenders. Keep the clone of the provider handle, instead of depending on above methods.
 
+- *Breaking* - `PeriodicReader` Updates
+
+  `PeriodicReader` no longer requires an async runtime by default. Instead, it now creates its own background thread for execution. This change allows metrics to be used in environments without async runtimes.
+
+  For users who prefer the previous behavior of relying on a specific `Runtime`, they can do so by enabling the feature flag **`experimental_metrics_periodicreader_with_async_runtime`**.
+
+  Migration Guide:
+
+1. *Default Implementation, requires no async runtime* (**Recommended**)
+   The new default implementation does not require a runtime argument. Replace the builder method accordingly:  
+   - *Before:* 
+     ```rust
+     let reader = PeriodicReader::builder(exporter, runtime::Tokio).build();
+     ```
+   - *After:*
+     ```rust
+     let reader = PeriodicReader::builder(exporter).build();
```

Comment:
OTLP Exporter has to default to HTTP and should use request-blocking client as default for this to work.
Add a todo here, to talk about what the user should be doing if they need to use tonic, similar to comments from https://github.com/open-telemetry/opentelemetry-rust/pull/2394